packages/eslint-plugin-mavenlint/rules/use-flux-standard-actions.js

```diff
@@ -14,8 +14,12 @@ module.exports = {
         const path = context.getFilename();
         if (!path.includes('action-creators')) { return; }
 
-        // Is this return value an Object literal?
         const statement = node.argument;
+
+        // Is the return statement empty
+        if (!statement) { return; }
+
+        // Is this return value an Object literal?
```

Comment:
Ooo, good catch. The line below is definitely trying to proceed only if the return value is an object literal, which no return value definitely is not.

Should we change the line below to:

```js
if (!(statement && statement.type === 'ObjectExpression')) { return; }
```

That seems to capture the original intent a little better and takes care of the issue here.

Reply:
I'm happy to make that change.